oak-blob-cloud/src/main/java/org/apache/jackrabbit/oak/blob/cloud/s3/S3Constants.java

```diff
@@ -107,11 +107,23 @@ public final class S3Constants {
      */
     public static final String S3_ENCRYPTION_SSE_KMS = "SSE_KMS";
 
+    /**
+     *  Constant to set SSE_KMS encryption.
```

Comment:
I think this comment is wrong.
```suggestion
     *  Constant to set SSE_C encryption.
```